app/controllers/v3/service_brokers_controller.rb

```diff
@@ -59,6 +59,7 @@ def create
       space = Space.where(guid: message.space_guid).first
       unprocessable_space! unless space && permission_queryer.can_read_from_space?(space.guid, space.organization_guid)
       unauthorized! unless permission_queryer.can_write_space_scoped_service_broker?(space.guid)
+      suspended! unless permission_queryer.is_space_org_active?(space.guid)
```

Comment:
Add tests to spec/request/service_brokers_spec.rb:
- post '/service_brokers'
- patch '/service_brokers/:guid'
- delete '/service_brokers/:guid'

Reply:
- patch '/service_brokers/:guid'

  When org is suspended, the test about permission for update endpoint is already implemented, so I adjust the response message of the calling function `responses_for_org_suspended_space_restricted_update_endpoint` in `spec/support/space_restricted_response_generators.rb`

- post '/service_brokers'
 
  Adjust `responses_for_org_suspended_space_restricted_create_endpoint` function in `spec/support/space_restricted_response_generators.rb`

- delete '/service_brokers/:guid'

  Adjust `responses_for_org_suspended_space_restricted_delete_endpoint` in `spec/support/space_restricted_response_generators.rb`